sappho/layers.py

```diff
@@ -1,5 +1,8 @@
 import pygame
 
+PY3 = sys.version_info[0] == 3
+_range = range if PY3 else xrange
```

Comment:
The typical convention for avoiding overriding names is like `range_` because it distinguishes it from `_private_methods` and the like.